src/main/java/beam/utils/TravelTimeCalculatorHelper.java

```diff
@@ -68,6 +68,28 @@ private int getOffset(double time){
         return result;
     }
 
+    public static Map<String, double[]> GetLinkIdToTravelTimeAvgArray(Collection<? extends Link> links, TravelTime travelTime1, TravelTime travelTime2, int maxHour) {
```

Comment:
Just quick note, I would reuse `GetLinkIdToTravelTimeArray` and implement `GetLinkIdToTravelTimeAvgArray` on top of that. Basically get GetLinkIdToTravelTimeArray from both and average them.